scripts/package-build/aws-gwlbtun/package.toml

```diff
@@ -1,9 +1,32 @@
 [[packages]]
 name = "aws-gwlbtun"
-commit_id = "f78058a"
+commit_id = "c1fd48b"
 scm_url = "https://github.com/aws-samples/aws-gateway-load-balancer-tunnel-handler"
 
 
+# Newest 'asw-gwlbtun' requires the specific boots version (1.83) (available in the trixie)
+pre_build_hook = """
+echo "deb http://deb.debian.org/debian trixie main" | sudo tee /etc/apt/sources.list.d/trixie.list
+
+sudo bash -c 'cat > /etc/apt/preferences.d/trixie-pin <<EOF
```

Comment:
Is it worth the trouble? Are there dramatic improvements in that version, or is the old version going to stop working soon?

Reply:
> Are there dramatic improvements in that version, or is the old version going to stop working soon?

@dmbaturin This is AWS requirement https://github.com/aws-samples/aws-gateway-load-balancer-tunnel-handler/blob/c1fd48b7b05879b199ebf1beeeacf0442b3c4ad2/CMakeLists.txt#L7